doc/running.md

```diff
@@ -113,3 +113,13 @@ Go to http://localhost:8000/
 ```sh
 make test
 ```
+
+or directly using `pytest`:
+
+```sh
+DJANGO_SETTINGS_MODULE=controlpanel.settings.test pytest --color=yes
+```
+
+**NOTE** Set the `DJANGO_SETTINGS_MODULE` is important or otherwise you
+may accidentally run the tests with the `development` settings with
+unpredictable results.
```

Comment:
The `DJANGO_SETTINGS_MODULE` used by `pytest` is configured in `/setup.cfg`, you shouldn't have to set it manually - is that not working?

Reply:
Also, you don't need to pass `--color=yes`, colours are automatically enabled if the terminal is capable of showing them

Reply:
When I got this problem I found out that yes this is configured in `./setup.cfg` but in my case I also had that `DJANGO_SETTINGS_MODULE` set in my environment variable which unfortunately (in this case!) takes precedence against the `setup.cfg`.

This is by design and makes a lot of sense but I imagined it could happen to others following this documentation as one of the first things at the top of the page is to set `DJANGO_SETTINGS_MODULE` to `*.development`. I just thought having a simple note here could save someone else some time in the future.

Reply:
Regarding `--color=yes` I'll remove, I just copied whatever was in the `Makefile`.